gravitee-apim-portal-webui-next/projects/gravitee-dashboard/src/lib/components/chart/line-chart/line-chart.component.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, computed, inject, input } from '@angular/core';
+import { ChartConfiguration } from 'chart.js';
+import { BaseChartDirective } from 'ng2-charts';
+// eslint-disable-next-line import/no-unresolved
+import 'chartjs-adapter-date-fns';
+
+import { LineConverterService } from './converter/line-converter.service';
+import { TimeSeriesResponse } from '../../widget/model/response/time-series-response';
+
+export type LineType = 'line';
+
+@Component({
+  selector: 'gd-line-chart',
+  imports: [BaseChartDirective],
+  templateUrl: './line-chart.component.html',
+  styleUrl: './line-chart.component.scss',
+})
+export class LineChartComponent {
+  type = input<LineType>('line');
+  option = input<ChartConfiguration<LineType>['options']>(this.getDefaultOptions());
+  data = input.required<TimeSeriesResponse>();
+
+  public readonly dataFormatted = computed(() => {
+    const chartData = this.converter.convert(this.data());
+
+    chartData.datasets.forEach(dataset => {
+      dataset.tension = 0.4;
+      dataset.fill = false;
+    });
+
+    return chartData;
+  });
+  private readonly converter = inject(LineConverterService);
+
+  private getDefaultOptions(): ChartConfiguration<LineType>['options'] {
+    return {
+      responsive: true,
+      maintainAspectRatio: false,
+      plugins: {
+        legend: {
+          display: true,
+          position: 'top',
```

Comment:
https://www.figma.com/design/kmHy6HValvzVi5LLfciBCt/Analytics?node-id=10310-24450&t=U6mG5ukgJNHKFukQ-4

```suggestion
          position: 'bottom',
```

Reply:
Oops! Thanks for flagging. Updated in [1aa5432](https://github.com/gravitee-io/gravitee-api-management/pull/14199/commits/1aa5432d45a504240c82fbdc3c590ce6b7cc032a).